contributors/devel/godep.md

```diff
@@ -156,6 +163,13 @@ transitively), you'll also need to update deps there:
 ./hack/update-staging-godeps.sh
 ```
 
+Commit the changes generated by the above script:
+
+```sh
+git add .
```

Comment:
is it needed to explicitly mention the commands? Maybe just asking to commit the changes is enough? :thinking:

Reply:
or `git commit --amend`? is there a best practice here - both changes should go into one commit? :)

Reply:
I would suggest that we don't call out specific commands. Sometimes you want to have them be in the same commit, other times not.

Reply:
> I would suggest that we don't call out specific commands. Sometimes you want to have them be in the same commit, other times not.

:+1: done.